chart/amplication/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: amplication
+description: A Helm chart for Amplication.
+
+dependencies:
+- name: postgresql
+  alias: "postgres"
+  version: "10.3.12"
+  repository: "https://charts.bitnami.com/bitnami"
+  condition: postgres.install
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 1.16.0
```

Comment:
This should be 0.5.0 IIUC
Also, how can this be automatically updated?

Reply:
The app version typically corresponds with the Docker tags. When I was digging through your Docker Hub I didn't see any versioning happening. (most of it was 'latest').

Updating this version, and the chart version usually happen during a build pipeline that's tied to your Docker image(s) builds. I see you have [tagging happening](https://github.com/amplication/amplication/releases/tag/v0.5.0), do these tags also build out tagged images? 

For now I'll update the `appVersion` to `0.5.0`, but I think it's something to consider automating in the future; and that piece is usually custom to your pipelines.

Reply:
https://github.com/amplication/amplication/pull/1276/commits/5a3a4fd5c65d6efadab8d2a00bfd0a5efed69d5c